android/app/src/main/java/com/ledger/live/MainApplication.java

```diff
@@ -55,7 +54,6 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
-            new FingerprintAuthPackage(),
```

Comment:
mmh this code shouldn't be removed right?

Reply:
It was duplicated, it's still imported and listed in the packages in the current file.